### ISSUE TITLE
Isotope pattern preview window crashes

### ISSUE THREAD

**Issue by tomas-pluskal** [mzmine/mzmine2#687]
The Isotope pattern preview window seems to have difficulty with simple chemical formulas like "C1": 

```
[21:17:29|WARNING|net.sf.mzmine.util.FormulaUtils]: Formula invalid! Element(s) C1 do not exist.
[21:17:29|FINE|net.sf.mzmine.modules.tools.isotopepatternpreview.IsotopePatternPreviewDialog]: Invalid input or Element == "" or invalid elements.
[21:17:29|FINE|net.sf.mzmine.modules.tools.isotopepatternpreview.IsotopePatternPreviewDialog]: updateParameters() failed due to invalid input.
[21:17:29|WARNING|net.sf.mzmine.modules.tools.isotopepatternpreview.IsotopePatternPreviewDialog]: updateWindow() failed. Could not update parameters or parameters are invalid.
Please check the parameters.
```

**Comment by tomas-pluskal** [comment]
@SteffenHeu

**Comment by SteffenHeu** [comment]
I will look into it as soon as I can. I'm sure it's a simple check that fails, so it should be a quick fix. I am having trouble setting up java 12 and and gradle though and I am leaving for a research internship tomorrow, so I'm quite busy at the moment.
I am confident, that i can do a fix within the next couple of days, though.

**Comment by tomas-pluskal** [comment]
Fixed in version 2.51
